ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.cpp

```diff
@@ -183,7 +183,7 @@ void Geotriggers::handleGeotriggerNotification(GeotriggerNotificationInfo* geotr
     m_currentFeaturesEntered[geotriggerName].removeAll(featureName);
   }
 
-  displayInfoChanged();
+  emit displayInfoChanged();
```

Comment:
Missing keyword `emit` for signals. (I've missed something about the use of `emit`?)

Reply:
A comment about this one: `emit` is an empty `#define`, it does nothing. Then it's not an error to not use `emit`.

But I think we should use `emit`:
- to be consistent with the existing code
- to improve readability of signal emitting

I guess if we want to remove `emit` keyword, we should create an issue to talk about this guideline and remove `emit` everywhere in the code.

See https://www.kdab.com/nailing-13-signal-slot-mistakes-clazy-1-3/ about this clazy warning.

Reply:
We should keep using `emit`, even if it technically does nothing. It adds a lot to the readability of the code, and informs people that we are emitting a signal as opposed to calling a function.

I agree with the clazy doc:
>11. incorrect-emit
>For readability purposes you should always use emit (or Q_EMIT) when calling a signal. Conversely, you should not use those macros when calling something other than a signal.